rpcs3/Crypto/unpkg.cpp

```diff
@@ -203,7 +203,7 @@ bool pkg_install(const std::string& path, atomic_t<double>& sync)
 	// Get full path and create the directory
 	const std::string dir = Emulator::GetHddDir() + "game/" + install_id + '/';
 
-	if (!fs::create_dir(dir))
```

Comment:
Ok, need to change also the logic to set was_full variable, because create_path can't fail on existing path.

Reply:
Changed, untested.